app/models/concerns/omniauthable.rb

```diff
@@ -58,13 +58,14 @@ def user_params_from_auth(auth)
       email_is_verified = auth.info.verified || auth.info.verified_email || assume_verified
       email             = auth.info.verified_email || auth.info.email
       email             = email_is_verified && !User.exists?(email: auth.info.email) && email
+      display_name      = auth.info.full_name || [auth.info.first_name, auth.info.last_name].join(' ')
```

Comment:
@ineffyble Thanks for the work ! Just one comment : If the full_name is not provided by SAML, the user willl be registered will fullname " " (space char), since `[nil, nil].join(' ') == ' '`. I fixed my previous suggestion by adding `strip` there https://github.com/tootsuite/mastodon/issues/6535#issuecomment-367932856

@Gargron Dont play lottery with those numbers ;)

Reply:
Do `.compact.join` then